examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml

```diff
@@ -66,6 +66,16 @@ spec:
           - name: echo
             image: ubuntu
             script: 'echo file exists'
+    - name: echo-file-missing
```

Comment:
i think it would be worth adding a couple comments or changing the names of the pipelinetasks to reflect what we are trying to cover here, which i think is:

1. using a task result which causes the when expression to fail (echo-file-missing)
2. using a pipeline param which causes the when expression to fail (task-should-be-skipped)

This example doesnt seem to include any cases where the when expression succeeds 🤔  cant remember if there was another exmaple? i think we should add that as well if not

but we dont have to do that in this pr!

Reply:
sounds good, will update the names/add comments 

`echo-file-exists` is an example using when expressions with results and succeeds -- it uses the same result `$(tasks.check-file.results.exists)` and evaluates to true:
https://github.com/tektoncd/pipeline/blob/18dd0af6abf304c18b1484c52cf3b75c7e985160/examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml#L59-L78

`create-file` is the example that uses when expressions with params and succeeds:
https://github.com/tektoncd/pipeline/blob/18dd0af6abf304c18b1484c52cf3b75c7e985160/examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml#L16-L31

Reply:
whoops, sorry i missed that @jerop !